boilerplate/openshift/golang-osd-operator/standard.mk

```diff
@@ -332,3 +333,27 @@ opm-build-push: docker-push
 .PHONY: ensure-fips
 ensure-fips:
 	${CONVENTION_DIR}/configure-fips.sh
+
+.PHONY: migrate-tools
+migrate-tools:
+	# Installing controller-gen v0.8.0
+	go install sigs.k8s.io/controller-tools/cmd/controller-gen@${CONTROLLER_GEN_VERSION}
+	mv ${GOBIN}/controller-gen ${GOBIN}/controller-gen-${CONTROLLER_GEN_VERSION}
+
+	# Installing openapi-gen v0.23.0
+	go install k8s.io/code-generator/cmd/openapi-gen@${OPENAPI_GEN_VERSION}
+	mv ${GOBIN}/openapi-gen ${GOBIN}/controller-gen-${CONTROLLER_GEN_VERSION}
```

Comment:
For maximum portability, instead of installing these locally, I would recommend making the `migrate` target run `generate` through `container-make`. Suggestion below.